test/functional/tests/lazy_writes/wb_throttling.py

```diff
@@ -0,0 +1,153 @@
+#
+# Copyright(c) 2019 Intel Corporation
+# SPDX-License-Identifier: BSD-3-Clause-Clear
+#
+
+import pytest
+import time
+import random
+from datetime import timedelta
+
+from test_utils.size import Size, Unit
+from core.test_run import TestRun
+from storage_devices.disk import DiskType, DiskTypeSet, DiskTypeLowerThan
+from test_tools import fs_utils
+from test_tools.fio.fio import Fio
+from test_tools.fio.fio_param import ReadWrite, IoEngine
+from test_tools.iostat import IOstat
+from test_utils.os_utils import (
+    kill_all_io,
+    set_wbt_lat,
+    get_wbt_lat,
+    get_dut_cpu_number,
+)
+from api.cas import casadm
+from api.cas.cache_config import (
+    CacheMode,
+    CacheModeTrait,
+    CleaningPolicy,
+    FlushParametersAcp,
+    SeqCutOffPolicy,
+    CacheLineSize,
+    Time,
+)
+from test_tools.blktrace import BlkTrace, BlkTraceMask, ActionKind, RwbsKind
+
+
+runtime = 60 * 60 * 24 * 30
+
+
+@pytest.mark.require_disk("cache", DiskTypeSet([DiskType.optane, DiskType.nand]))
+@pytest.mark.require_disk("core", DiskTypeLowerThan("cache"))
+def test_wb_throttling():
+    """
+        title: Test CAS with write back throttling enabled on exported object
+        description: |
+          Fill cache with data, run intensive IO (rwmix=70) with occasionals trims and wait for
```

Comment:
occasional

Reply:
"wait for panic" is not an expected behaviour ;) especially looking at the pass criteria. I suggest removing everything after "and" including "and" :)